tests/unittest/executor/test_proxy_worker_startup.py

```
@@ -0,0 +1,481 @@
+
+    def on_poll(poll_count):
+        if poll_count >= 5:
+            polled_repeatedly.set()
+
+    proxy = _make_proxy(monkeypatch, futures=[future], on_poll=on_poll)
+    result = {}
+
+    def start_workers():
+        try:
+            proxy._start_executor_workers({})
+        except BaseException as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `BaseException`

(BLE001)